rules/windows/builtin/win_user_added_to_local_administrators.yml

```diff
@@ -18,7 +18,7 @@ detection:
     selection:
         EventID: 4732
     selection_group1:
-        GroupName: 'Administrators'
+        GroupName|startswith: 'Administr'
```

Comment:
What do you think of this approach?

Reply:
Hi,
Good, much more efficient.

Reply:
it is `TargetUserName|startswith: 'Administr'`